### ISSUE TITLE
Missleading report, if path to provider-metadata.json is wrong security.txt

### ISSUE THREAD

**Issue by mfd2007** [csaf-poc/csaf_distribution#261]
"version": "0.9.5-1-g490fe4a",

If the CSAF entry in `security.txt` points to a wrong/not available path, the following error is returned:

```
{
          "num": 7,
          "description": "provider-metadata.json",
          "messages": [
            {
              "type": 1,
              "text": "\"https://csaf-provider-1.csaf/.well-known/csaf/provider-metadata.json\" not found."
            },
            {
              "type": 1,
              "text": "https://csaf-provider-1.csaf/.well-known/security.txt failed to load."
            },
            {
              "type": 1,
              "text": "\"https://csaf.data.security.csaf-provider-1.csaf\" not found."
            },
            {
              "type": 2,
              "text": "No valid provider-metadata.json found."
            },
            {
              "type": 2,
              "text": "STOPPING here - cannot perform other checks."
            }
          ]
        },
```

My expectation would be that the address, that is referenced in the security.txt, is also logged as not available.

**Comment by s-l-teichmann** [comment]
This could also be addressed by PR #260 [here](https://github.com/csaf-poc/csaf_distribution/pull/260/files#diff-113660444d09f68a1dcf1d2f5e1363630b30a96af560762b40037f8228ad6a01R140).

**Comment by mfd2007** [comment]
Yes, is already fixed (my fault).
